### ISSUE TITLE
Will this plugin ever be added to the Lidarr master branch??

### ISSUE THREAD

**Issue by nikmax42** [ta264/Lidarr.Plugin.Deemix#14]
In other issues you [spoke](https://github.com/ta264/Lidarr.Plugin.Deemix/issues/1#issuecomment-1220629590) that this repository was created for development and plugin is planned to be added in the official Lidarr, right? But it's still not in the official Lidarr, at least in the docker container I installed. Is Lidarr developers decided to not add this plugin to the official branch? Is it available on develop or nightly Lidarr branch?

**Comment by ceminess** [comment]
You need to use the plugins branch.  I use docker image hotio/lidarr:pr-plugins.

**Comment by mynameisbogdan** [comment]
No, Lidarr master branch will be without plugins added by default.
